lib/crypto/src/crypto.erl

```diff
@@ -744,28 +807,29 @@ enable_fips_mode(Enable) ->
                         case application:get_env(?MODULE, var_name(OldState)) of
                             undefined ->
                                 %% Make the next state change fast:
-                                OldCs = application:get_env(?MODULE,?CURVES), 
+                                OldCs = application:get_env(?MODULE, ?CURVES),
                                 application:set_env(?MODULE, var_name(OldState), OldCs);
                             _ ->
                                 ok
                         end,
                         %% Now rebuild the list by hard work:
-                        application:unset_env(?MODULE, ?CURVES), % This will force a re-build
+                        application:unset_env(?MODULE, ?CURVES),
```

Comment:
Strange choice, it seems.

Reply:
What is happening here is that the comments 
`% This will force a re-build` and `%% Re-build curves cache. Not strictly needed here.` are seen as a single comment block.
This means that `% This will force a re-build` is no longer seen as a trailing comment, so it is moved to live with  `%% Re-build curves cache. Not strictly needed here.` instead of moved to the line above, as all trailing comments usually are.

I have opened an issue for this https://github.com/WhatsApp/erlfmt/issues/168

Reply:
We have found that this is a bug, good catch :D